massifs/mmriver.go

```diff
@@ -0,0 +1,246 @@
+package massifs
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"fmt"
+
+	"github.com/datatrails/go-datatrails-common/azblob"
+	commoncbor "github.com/datatrails/go-datatrails-common/cbor"
+	"github.com/fxamacker/cbor/v2"
+
+	commoncose "github.com/datatrails/go-datatrails-common/cose"
+	"github.com/datatrails/go-datatrails-common/logger"
+	"github.com/datatrails/go-datatrails-merklelog/mmr"
+)
+
+// MMRIVER COSE Receipts to accompany our COSE MMRIVER seals
+
+type MMRiverInclusionProof struct {
+	Index         uint64   `cbor:"1,keyasint"`
+	InclusionPath [][]byte `cbor:"2,keyasint"`
+}
+
+type MMRiverConsistencyProof struct {
+	TreeSize1        uint64   `cbor:"1,keyasint"`
+	TreeSize2        uint64   `cbor:"2,keyasint"`
+	ConsistencyPaths [][]byte `cbor:"3,keyasint"`
+	RightPeaks       [][]byte `cbor:"4,keyasint"`
```

Comment:
minor: again in terms of accumulator term.

Personally I really like the term accumulator. 

the fact that it is the list of the peaks from the rhs is a nice property of mmrs?